Tests.Integration.Presentation.Web/Organizations/OrganizationTest.cs

```diff
@@ -103,5 +104,31 @@ public async Task Cannot_Create_Organization_Of_Type(OrganizationRole role, Orga
                 Assert.Equal(HttpStatusCode.Forbidden, result.StatusCode);
             }
         }
+
+        [Fact]
+        public async Task Can_Get_Organizations_Filtered_By_Cvr_Or_Name()
+        {
+            //Arrange
+            var login = await HttpApi.GetCookieAsync(OrganizationRole.GlobalAdmin);
+            var nameOrg1 = A<string>();
+            var cvrOrg1 = (A<int>() % 9999999999).ToString("D10");
+            const AccessModifier accessModifier = AccessModifier.Public;
+
+            //Act - perform the action with the actual role
+            await OrganizationHelper.SendCreateOrganizationRequestAsync(TestEnvironment.DefaultOrganizationId, nameOrg1, cvrOrg1, OrganizationTypeKeys.Kommune, accessModifier, login);
```

Comment:
remember to use "using" for these responses. Otherwise we will have an open connection which will not be deleted before 2.nd gc run (rarely) and that actually causes instability